type-definitions/most.d.ts

```diff
@@ -90,11 +93,14 @@ declare interface DisposeFn {
   (): void|Promise<any>;
 }
 
-export function create<A>(f: (add: (a:A) => any, end: (x:any) => any, error: (e:Error) => any) => void|DisposeFn): Stream<A>;
+export function create<A>(f: (add: (a: A) => any, end: (x: any) => any, error: (e: Error) => any) => void|DisposeFn): Stream<A>;
 export function just<A>(a: A): Stream<A>;
 export function of<A>(a: A): Stream<A>;
+export function empty(): Stream<any>;
+export function never(): Stream<any>;
 export function from<A>(as: Iterable<A>): Stream<A>;
 export function periodic<A>(period: number, a?: A): Stream<A>;
+export function fromEvent(event: string, target: any, useCapture?: boolean): Stream<Event>;
```

Comment:
I should never have conflated the two in the first place, but for now, we're stuck with my unfortunate, dual-purpose `fromEvent` :)

Given that, I do think it's important to provide the extra guidance of only allowing `useCapture` when `target` is an `EventTarget`.  While `fromEvent('blah', anEventEmitter, true)` isn't harmful, it also doesn't do anything useful beyond `fromEvent('blah', anEventEmitter)`.

We should also try to help people discover and use [`@most/dom-event`](https://github.com/mostjs/dom-event) instead of `fromEvent` for DOM events whenever possible.